v63003/outref/gtm4212.txt

```diff
@@ -0,0 +1,20 @@
+# Backing up Default to length 230 path (length of temp file is 24, so total path is 254)
+##TEST_AWKDB file ##TEST_PATH##/mumps.dat backed up in file ##TEST_PATH##[a/]*mumps.dat
```

Comment:
Can we have the actual full path name instead of TEST_AWK. That would help confirm the actual length used. Same with other such lines of output.